apps/condo/domains/common/components/tasks/TaskProgress.tsx

```diff
@@ -166,17 +170,22 @@ export const TaskProgressTracker: React.FC<ITaskProgressTrackerProps> = ({ task
     }, [record, removeTaskFromUI])
 
     useEffect(() => {
-        if (record && record.status !== TASK_PROCESSING_STATUS) {
+        const recordStatus = get(record, 'status')
+        if (record && recordStatus !== TASK_PROCESSING_STATUS) {
```

Comment:
What the difference? One extra line of code…

Reply:
you are right